workflow/controller/operator.go

```
@@ -2387,19 +2387,19 @@ func generateNodeName(name string, index int, desc interface{}) string {
 func expandSequence(seq *wfv1.Sequence) ([]wfv1.Item, error) {
 	var start, end int
 	var err error
-	if seq.Start != "" {
-		start, err = strconv.Atoi(seq.Start)
+	if seq.Start != nil {
+		start, err = strconv.Atoi(seq.Start.String())
```

`IntString` does have `InvVal`, but that method swallows errors